weatherman.py

```diff
@@ -1,4 +1,58 @@
 """
 weatherman is a software that generates reports about the past weather of murree
 """
-# this file will contain code in the future
+import re
+import sys
+
+from modules.extreme_year import extreme_temperatures_year as t1
+
+
+def main():
+    """
+    this is a driver function for weatherman.
+    :return:
+    """
+    parameters = sys.argv[1:]
```

Comment:
Can we use https://docs.python.org/3/library/getopt.html#module-getopt for parsing the params?

Reply:
done